modules/cloud/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
-        <jcloud.version>1.9.0</jcloud.version>
+        <jcloud.version>2.0.0</jcloud.version>
```

Comment:
Just wondering, why not that add another property, like `<jcloud.compute.version>1.9.3</jcloud.compute.version>`, and use it instead of duplicated `1.9.3` literals below? And it will also be auto-documented, at least it will be clear what is for.

Also if `jcloud.version` property is not used, I guess, it then could be removed from the `properties` all together.

What do you think?